lang/llvm-6.0/Portfile

```diff
@@ -340,6 +346,17 @@ if {[lsearch -exact $PortInfo(depends_build) port:cctools] != -1} {
         -DCMAKE_LIBTOOL=${prefix}/bin/libtool
 }
 
+set min_major_sys_py 11
+platform darwin {
+    # requires a newer python than the system default
+    if {${os.major} < ${min_major_sys_py}} {
+        depends_build-append port:python${py_ver_nodot}
+        depends_lib-append   port:python${py_ver_nodot}
```

Comment:
The `depends_build` here is redundant here and should be removed. It is completely covered by `depends_lib`.

Reply:
indeed. will fix, thanks.